desktop/renderer-app/src/pages/LoginPage/index.tsx

```diff
@@ -103,20 +111,37 @@ export const LoginPage = observer(function LoginPage() {
     const serviceURL = i18n.language.startsWith("zh") ? SERVICE_URL_CN : SERVICE_URL_EN;
 
     function renderButtonList(): React.ReactNode {
-        return (
-            <>
-                <LoginButton
-                    provider="wechat"
-                    text={i18n.t("login-wechat")}
-                    onLogin={handleLogin}
-                />
-                <LoginButton
-                    provider="github"
-                    text={i18n.t("login-github")}
-                    onLogin={handleLogin}
-                />
-            </>
-        );
+        if (process.env.FLAT_REGION === "America") {
+            return (
+                <>
+                    <LoginButton
+                        provider="google"
+                        text={i18n.t("login-google")}
+                        onLogin={handleLogin}
+                    />
+                    <LoginButton
+                        provider="github"
+                        text={i18n.t("login-github")}
+                        onLogin={handleLogin}
+                    />
+                </>
+            );
+        } else {
+            return (
+                <>
+                    <LoginButton
+                        provider="wechat"
+                        text={i18n.t("login-wechat")}
+                        onLogin={handleLogin}
+                    />
+                    <LoginButton
+                        provider="github"
+                        text={i18n.t("login-github")}
+                        onLogin={handleLogin}
+                    />
```

Comment:
The same code. 
https://github.com/netless-io/flat/pull/1465/files#diff-dc929394ba8846e5e8b066ba1414a6fb3384023061696bbb137144e136decdcaR122-R126

Reply:
GitHub login should exist when the FLAT_REGION value is America

Reply:
I guess this...

```jsx
if (isUS) {
  return <>
    <Google />
    <GitHub />
  </>
} else {
  return <>
    <WeChat />
    <GitHub />
  </>
}
```

...is more maintainable than

```jsx
return <>
  {isUS ? <Google /> : <WeChat />}
  <GitHub />
</>
```